_includes/markdown/Performance.md

```diff
@@ -0,0 +1,374 @@
+
+Performance should be a consideration throughout the entire engineering process. Significant performance gains can be achieved through attention to detail and following the best practices documented below.
+
+
+<h2 id="caching" class="anchor-heading">Caching {% include Util/link_anchor anchor="caching" %} {% include Util/top %}</h2>
+
+Caching is a key aspect in reaching optimal performance both from a server and browser optimisation perspective, below are caching approaches:
+
+*   Ensure all static assets have cache busters in the form of either a version or fingerprint in the filename or unique query string in the URL. This needs to be implemented along with a cache-control header max-age of at least 1 month but optimally 1 year.
+*   A CDN is highly recommended, most popular CDN’s may offer page caching, browser caching and image optimizations.
+*   [Page caching](https://10up.github.io/Engineering-Best-Practices/php/#page-caching) should be set up, a lot of common hosting companies including WP Engine and WordPress VIP offer full page caching.
+*   [Object caching](https://10up.github.io/Engineering-Best-Practices/php/#the-object-cache) should be leveraged where applicable on the server.
+*   All projects should have a [manifest file](https://web.dev/add-manifest/) to download static assets that will be rarely updated. The [manifest.json is included in 10ups WP Scaffold](https://github.com/10up/wp-scaffold/blob/trunk/themes/10up-theme/manifest.json) and will require custom configuration.
+
+<h2 id="images-and-media" class="anchor-heading">Images and Media {% include Util/link_anchor anchor="images-and-media" %} {% include Util/top %}</h2>
+
+*   Images should be optimized for Next-Gen formats. JPEG 2000, JPEG XR, and [WebP](https://developers.google.com/speed/webp) are image formats that have superior compression and quality characteristics compared to their older JPEG and PNG counterparts.
+*   Crop images appropriately, you do not need to create a crop for every size but in some cases a few extra crops to handle mobile proportions can be useful.
+*   Images should be served using srcset so that smaller sizes can be shown for smaller viewports.
+*   All images implemented through code should [contain a width and height attribute](https://web.dev/optimize-cls/#images-without-dimensions) this is especially important in avoiding Content Layout Shift issues.
+*   Assets in particular images, should be served through CDN
+*   Images, Videos and iFrames should be lazy loaded. Please note that [WordPress will handle browser level lazy loading](https://make.wordpress.org/core/2021/02/19/lazy-loading-iframes-in-5-7/) using [native lazy load](https://web.dev/native-lazy-loading/). In order for this to take effect the width and height should be set on the tag. 
```

Comment:
@brentvr we should mention something here about using the `facade` pattern for loading iframes: https://web.dev/third-party-facades/